### ISSUE TITLE
Set a default env variable to check if running under local debug server

### ISSUE THREAD

**Issue by jacintoArias** [logandk/serverless-wsgi#65]
I do not know if this is already implemented but I have not found it in the docs. My apologies in advance If it can be done easier. 

I find myself including a serverless env variable reading an `opts` argument to check if my code is running locally to disable some functionalities (logs, remote module loading...)

```
environment:
  DEBUG_SERVER: ${opt:debugServer, 'false'}
```

It would be nice if this were built in transparently from serverless via (werkzeug or the wsgi handler), I have checked the environment of the lambda function and I haven't seen anything like this.

I can help with this if you agree and confirm the proper design.

PS: Thanks for your amazing work, you saved us a lot of time sir.

**Comment by sippeangelo** [comment]
https://github.com/logandk/serverless-wsgi/blob/46550c766dccebb0e74e8236ea9af34bd39a5e5c/serve.py#L34

According to this line, it should set the IS_OFFLINE environment variable when running locally through the 'serve' command, but I really can't get it to work. It doesn't seem like it propagates down to my code's environment and the variable is missing completely...

Is it only me? Does this method work for you?

**Comment by sippeangelo** [comment]
I now found that if I _manually_ set the IS_OFFLINE variable to something random, like "dev", this happens:
```
(UpdateService) % sls wsgi serve
IS_OFFLINE = dev
 * Running on http://localhost:5000/ (Press CTRL+C to quit)
 * Restarting with stat
IS_OFFLINE = True
 * Debugger is active!
 * Debugger PIN: 234-613-447
```

So the environment variable isn't there upon first launch, but then the Werkzeug reloader takes over and reloads my code, it's suddenly overwritten with the correct value! What could be the cause of this?

**Comment by logandk** [comment]
@jacintoArias Thanks for the suggestion! As @sippeangelo pointed out, we already have an environment variable, `IS_OFFLINE` that is intended for this particular use case. It wasn't documented, so I added a mention of it to the README.

@sippeangelo You're right, the environment variable was being set after the application was imported, so checking it at import-time didn't work until the first reload. I've fixed it in the next release.
